Vision/README.md

```diff
@@ -1,6 +1,4 @@
-A New Focus for the W3C: Improving the Web’s Integrity
-
-# Our vision for the World-Wide Web
+## Introduction
```

Comment:
Retitled this as "introduction".  This still needs a rework itself to address #26, but that will be a separate effort.